### ISSUE TITLE
Undefined control sequence. \theoremstyle

### ISSUE THREAD

**Issue by davidwlindholm** [fmarotta/kaobook#205]
### Description
Compilation fails with the following error message:
```
newpxtext.sty (error):
line 563: Undefined control sequence. \theoremstyle
line 563: Missing \begin{document}. \theoremstyle{p

main.tex (warning):
: Unused global option(s): [fontsize,twoside].
: No file main.bbl.
line 94: non empty \uppertitleback ignored by \maketitle(scrbook) in `twoside=false' mode
line 94: non empty \lowertitleback ignored by \maketitle(scrbook) in `twoside=false' mode
: destination with the same identifier (name{page.1}) has been already used, duplicate ignored<to be read again> \relax l.141 \pagelayout{wide} % No margins
line 171: Empty bibliography
```

### Minimal Working Example
Just try to compile minimal_book from the latest release, using Texlive 

**Expected behavior:**
Compilation

**Actual behavior:**
No compilation

### Additional Information
This problem originally occurred on a fully updated installation of Manjaro Mate, and was confirmed on a completely fresh install of Manjaro KDE using Virtualbox. The bug persists in both TexStudio and TexMaker, which leads me to believe it must be a recent change in Texlive.
`tex --version` reports:
`TeX 3.141592653 (TeX Live 2021/Arch Linux)`
I also tried overwriting the release files with the latest updated versions from the repo, with no difference.

**Comment by nercisse** [comment]
I ran into the same issue on Ubuntu with TexLive, same version as yours. Although in my case I could continue the compilation process.

After a bit of investigation it seems the problem comes from a commented line in **newpxtext.sty**, at  **l.406**.

`        %\RequirePackage{amsmath,amsthm} % in case of theoremfont`


I just removed the `%` to make sure the amsmath and amsthm packages are used. I do not know why or when it's been commented out in TexLive, but it did the trick for me.
